### ISSUE TITLE
[Feature Request] Mass delete downloaded content whose titles are not in the library

### ISSUE THREAD

**Issue by labadidadee** [jobobby04/TachiyomiSY#72]
**Why/User Benefit/User Problem**
I tend to download a lot of manga for ease of reading but tend to reserve library slots for those I actually enjoy reading/wish to reread. Bc some sources have wonky content, I also migrate a lot. However, given that virtually everything in my library is downloaded, I end up having two copies of each title, some of which I don't even realize I have until I somehow end up on the same source I've migrated from and find all these downloaded content taking up my space. I think this is a nice feature to have to eliminate the hassle of removing each title manually in the tachiyomi folder. It's also near impossible/incredibly difficult to cross-check the downloaded content across different source folders and the tachiyomi lib every time you make the decision to delete a folder or not, especially if one has hundreds/thousands of library titles and nearly double that in downloaded content.

**What/Requirements**
Currently, there's a 'clear database' option which clears the history for manga not saved in the library. I imagine this to be a similar but a more thorough version of the toggle in the manner that it really clears the tachiyomi database of non-library content. It was mentioned on tachi discord that J2K already has this feature but I'm unsure given that I don't have J2K on my phone.

**Comment by jobobby04** [comment]
That should be possible in the latest preview build, now that I ported the J2k feature over to SY(unless I am misunderstanding what it does)

**Comment by labadidadee** [comment]
> That should be possible in the latest preview build, now that I ported the J2k feature over to SY(unless I am misunderstanding what it does)

> That should be possible in the latest preview build, now that I ported the J2k feature over to SY(unless I am misunderstanding what it does)

I looked at the preview build just now and it isn't what I'm looking for, I think. Delete non-existent, partially-downloaded chapters is somewhat it, but I don't want to delete read chapters given that I like to reread them offline. The feature I specifically want is the deletion of non-library downloaded content. Like, for example, I once downloaded x chapters from source y, but I migrated to source z and re-downloaded stuff from there. Given that the title from source y is no longer inside the library and I have another copy of the chapters from the source z, the downloaded content from it is only taking up unecessary redundant space. There are also cases when I download stuff before putting them in the library and end up not liking them so I don't put them in the library at all. There's virtually no way to get back to these titles if you forget to remove them immediately after reading because they're not added to the library. The only ways to delete those non-lib content are to a) happen to stumble across them again and delete them then, or b) manually delete titles from the tachi folder while cross checking if the library has them or not (down to the exact source). The first option is just impossible and the second one is just an inefficient way of doing things when one has thousands of downloaded titles (which I do). I hope this explanation is sufficient :)
